.github/workflows/python-package-shared.yml

```diff
@@ -36,7 +36,7 @@ jobs:
   integration:
     runs-on: ubuntu-latest
     concurrency:
-      group: labelbox-python-${{ inputs.test-env }}-${{ inputs.python-version }}
+      group: labelbox-python-${{ inputs.test-env }}-${{ inputs.python-version }}-integration
```

Comment:
why is this added, curious

Reply:
This ensures only 1 set of integration test is run per testing environment / python version